src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -22,12 +22,21 @@ import Foundation
 /// The `SecureStore` class implements the `Store` protocol that defines all required storage attributes.
 /// It uses an SQLite Database that still needs to be encrypted
 final class SecureStore: Store {
+
+	// MARK: - Static.
+
 	private let directoryURL: URL
 	private let kvStore: SQLiteKeyValueStore
+	private let localServerEnvironment: LocalServerEnvironment
```

Comment:
Just a suggestion:
I would prefer 'environment' as name because the setup is local but the servers are definitely not local.

Reply:
renamed